main.js

```diff
@@ -0,0 +1,10 @@
+$(document).ready(function(){
+  
+  $('button').on('click', function(el){
```

Comment:
You want to add the event listener to the form itself -- right now, this code is running each time you click the button. That means you don't have to enter anything in for the form to return the alert.